recipes/session-info2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "session-info2" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/session_info2-{{ version }}.tar.gz
+  sha256: bdb75885128333611e0cf4357f31a0b5869bcb7889b93251ed8571aa4e4ad0b6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }}
```

Comment:
```suggestion
    - python {{ python_min }}
```

Reply:
sure. maybe the maintainers update grayskull so it generates this code!

Reply:
WIP: https://github.com/conda/grayskull/issues/574 and https://github.com/conda/grayskull/pull/581